### PR TITLE
Resolve #12 -- Improve unzipping of client

### LINKED ISSUE #12: Set `"League-of-Legends-4-20"` as a class variable/const
You could set `"League-of-Legends-4-20"` as a class variable/const. Doesn't really matter as the code doesn't really change much.

_Originally posted by @danil179 in https://github.com/LeagueSandbox/LeagueSandboxAutoSetup/pull/10#discussion_r619888017_

### PR DESCRIPTION
**Pull request by Kipero**
* Add clientArchive as const, resolves #12 
* Add WaitForFile to wait for getting access to file as user is pasting it to installDirectory

Resolve #12

### CODE REVIEW

File: League Sandbox Auto Setup/Util/Utility.cs [hunk 1]
```diff
@@ -18,5 +19,25 @@ public static void CreateShortcut(string shortcutName, string shortcutPath, stri
...
+            {
+                try
+                {
+                    action();
+                    return;
+                }
+                catch (IOException e)
+                {
+                    // access error
+                    if (e.HResult != -2147024864)
```
Comment (danil179) [inline League Sandbox Auto Setup/Util/Utility.cs:36]: Is there a nicer way to write the magic number `-2147024864`?

**Review by danil179 — COMMENTED** [review]
Comment about a magic number.

File: League Sandbox Auto Setup/Util/Utility.cs (hunk 1 above)
Comment (Kipero) [reply League Sandbox Auto Setup/Util/Utility.cs:36]: Tried to get it through errlook.exe in Visual Studio tools, all solutions which I found in stackoverflow just based on this value, we could maybe set it as const "accessError"?

**Review by danil179 — APPROVED** [review]
Approved. Magic numbers introduced, but in a very restricted manner and there are no clear workarounds.

**Pull request merged**